sanity-check.js

```diff
@@ -0,0 +1,32 @@
+const dict = require('./data/dictionary.json');
+const params = require('./src/params.js');
+
+/**
+ * this transfer '_xxx_count' to 'xxx'
+ * @param {string} countName
+ */
+function parseNodeType(countName) {
+  return countName.substring(1, countName.length - '_count'.length);
+}
+const boardCountNodes = params.config.graphql.boardCounts.map(e => parseNodeType(e.graphql));
+const chartCountNodes = params.config.graphql.boardCounts.map(e => parseNodeType(e.graphql));
```

Comment:
should be `graphql.chartCounts` ?

Reply:
🤔  why? it's getting the node name from chartCounts config....

Reply:
Oh I get it....typo here !!